src/shared/schema/avl-consumer-subscriptions.schema.ts

```diff
@@ -11,12 +11,21 @@ export const avlConsumerSubscriptionSchema = z.object({
     heartbeatInterval: z.string(),
     initialTerminationTime: z.string(),
     requestTimestamp: z.string(),
-    producerSubscriptionIds: z.string(),
     heartbeatAttempts: z.number(),
     lastRetrievedAvlId: z.number(),
     queueUrl: z.union([z.literal(""), z.string().url()]),
     eventSourceMappingUuid: z.string(),
     scheduleName: z.string(),
+    queryParams: z.object({
+        boundingBox: z.number().array().min(4).max(4).optional(),
+        operatorRef: z.string().min(1).array().optional(),
+        vehicleRef: z.string().min(1).optional(),
+        lineRef: z.string().min(1).optional(),
+        producerRef: z.string().min(1).optional(),
+        originRef: z.string().min(1).optional(),
+        destinationRef: z.string().min(1).optional(),
```

Comment:
Would it be worth adding NMTOKEN validation like we do for the siri-vm downloader?
```
        boundingBox: createBoundingBoxValidation("boundingBox").optional(),
        operatorRef: createNmTokenArrayValidation("operatorRef").optional(),
        vehicleRef: createNmTokenValidation("vehicleRef").optional(),
        lineRef: createNmTokenValidation("lineRef").optional(),
        producerRef: createNmTokenValidation("producerRef").optional(),
        originRef: createNmTokenValidation("originRef").optional(),
        destinationRef: createNmTokenValidation("destinationRef").optional(),
```